sdk/trace-shaded-deps/src/main/java/io/opentelemetry/sdk/trace/internal/JcTools.java

```diff
@@ -42,5 +43,30 @@ public static long capacity(Queue<?> queue) {
     }
   }
 
+  /**
+   * Remove up to <i>maxExportBatchSize</i> elements from the {@link Queue} and hand to consume.
+   *
+   * @throws IllegalArgumentException consumer is {@code null}
+   * @throws IllegalArgumentException if maxExportBatchSize is negative
+   */
+  @SuppressWarnings("unchecked")
+  public static <SpanT> void drain(
```

Comment:
Let's remove span from this file, can just be `T` and `item`.

Reply:
the naming is just to improve readability here, as this method is mainly used here to pass Span type data only. If its a generic module/lib than "T" or "itemT"  will be a better choice. What do you think ?

Reply:
We'll want to refactor to use this for logs too, there isn't any logic specific to spans so it's more confusing to use the word here.

Reply:
Okay will update in next rev.